tests/redux/reducers/award/awardReducer-test.jsx

```diff
@@ -0,0 +1,420 @@
+                        action_type: null,
+                        federal_action_obligation: '-$14,274',
+                        modification_number: '860',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - RECOVERY TAS::89 0227::TAS - DEOBLIGATE SC ARRA FUNDING FROM PROJECT ENTITLED "NUCLEAR PHYSICS - NUCLEAR SCIENCE WORKFORCE"'
+                    }
+                ]
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactions).toEqual(action.transactions);
+        });
+    });
+
+    describe('SET_TRANSACTIONS_META', () => {
+        it('should save the returned award to redux state', () => {
+            const action = {
+                type: 'SET_TRANSACTIONS_META',
+                count: 13,
+                page: 1,
+                totalPages: 2
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactionMeta).toEqual(action.transactionMeta);
+        });
+    });
+
+    describe('SET_TRANSACTION_SORT', () => {
+        it('should set the sorting column and direction', () => {
+            const action = {
+                type: 'SET_TRANSACTION_SORT',
+                direction: 'desc',
+                field: 'modification_number'
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactionSort).toEqual(action.transactionSort);
+        });
+    });
+
+    describe('UPDATE_TXN_RENDER_HASH', () => {
+        it('should provide a unique hash id for the transaction render', () => {
+            const action = {
+                type: 'UPDATE_TXN_RENDER_HASH',
+                renderHash: _.uniqueId()
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.renderHash).not.toEqual(action.renderHash);
+        });
+    });
+
+    describe('UPDATE_TXN_GROUP_HASH', () => {
+        it('should provide a unique hash for the transaction group', () => {
+            const action = {
+                type: 'UPDATE_TXN_GROUP_HASH',
+                groupHash: _.uniqueId()
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.groupHash).not.toEqual(action.groupHash);
+        });
+    });
+
+    describe('RESET_AWARD_DATA', () => {
+        it('should reset state to the initial state', () => {
+            const action = {
+                type: 'RESET_AWARD_DATA'
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState).toEqual(initialState);
```

Comment:
This doesn't give us assurance that the reset Redux action works, since we are simply resetting the initial state back to the initial state...which means nothing is happening. You need to modify the Redux state to be something different from the initial state, perform the reset Redux action, and then test that the new Redux state matches the initial state.